packages/wine.rb

```diff
@@ -54,29 +78,39 @@ def self.install
       Dir.chdir "#{CREW_DEST_PREFIX}/bin" do
         FileUtils.ln_s 'wine64', 'wine'
       end
+      Dir.chdir 'wine64-build' do
+        system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+      end
+    else
+      system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     end
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
+    @wine_config_env = <<~'WINE_CONFIG_EOF'
+      # Wine configuration
+      WINEPREFIX="${XDG_CONFIG_HOME}"/.wine
+    WINE_CONFIG_EOF
+    IO.write("#{CREW_DEST_PREFIX}/etc/env.d/wine", @wine_config_env)
   end
 
   def self.postinstall
     puts
-    puts "Type 'wine explorer' to get started using the file explorer.".lightblue
+    puts "Type 'WINEPREFIX=#{@xdg_config_home}/.wine wine explorer' to get started using the file explorer.".lightblue
```

Comment:
Would it make sense to put this in a wrapper script?

Reply:
I don't think so. That's just there in case WINEPREFIX wasn't set.

We're talking about people who put their devices in dev mode. We don't need to mollycoddle everyone!

This should indicate to people that they need WINEPREFIX set. It's wine. It's not `gemacs`.

Reply:
Don't you realize everybody needs mollycoddled today? ;)